### PR TITLE
fix: Prototype changes

### PR DESCRIPTION
**Pull request by sdsantos**

### CODE REVIEW

File: lib/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -1,6 +1,19 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
     package="tech.relaycorp.relaydroid"
     >
 
+  <uses-permission android:name="android.permission.INTERNET" />
```
Comment (gnarea) [inline lib/src/main/AndroidManifest.xml:7]: Do we really need `android.permission.INTERNET` or is it a mistake?

**Review by gnarea — COMMENTED** [review]
I also created https://github.com/relaycorp/relaynet-endpoint-android/issues/40

File: lib/src/main/AndroidManifest.xml (hunk 1 above)
Comment (sdsantos) [reply lib/src/main/AndroidManifest.xml:7]: I believe we do need for PoWeb, but we can try taking it out and check if anything is blocked.

Comment (gnarea) [reply lib/src/main/AndroidManifest.xml:7]: Cool. Just created https://github.com/relaycorp/relaynet-endpoint-android/issues/43 to look into it.

**Review by gnarea — APPROVED** [review]

**Pull request merged**